### ISSUE TITLE
Upgrade serialport to latest stable

### ISSUE THREAD

**Issue by beriberikix** [spark/particle-dev#229]
See related spark/particle-cli#390. I also noticed we're using a [fork](https://github.com/spark/particle-dev/blob/master/package.json#L29) of node-serialport - do we know if we can move to the mainline version?

**Comment by suda** [comment]
We're using the fork because the `serialport` module didn't previously work well with Electron. We could try using the latest version and seeing if it's ok now.

**Comment by beriberikix** [comment]
Makes sense, thanks for the background. It looks like there are now separate, optimized builds specifically for Electron ✨
